### DISCUSSION TITLE
Be able to Preset on Directus Roles

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#18849] [Feature Requests]
### Summary

With (many) roles, it would be nice to show them on a specific order.
At least alphabetically or according to an added field `sort`.

ATM when you click on the Role field to choose the user's role, the list of roles is displayed in a cumbersome order.

When trying to create a preset on Directus Roles, only Directus Activity, Files and Users are offered.

It would be NTH also the possibility of creating a preset on Directus Roles

### Motivation

With more than twenty roles, it is better to show them in order

### Detailed Design

Add `Directus Roles` to the list of collections for a preset

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

None

### Alternatives

None

### Adoption Strategy

No impact. Immediate adoption

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
